config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/beans/Bundle.java

```diff
@@ -45,8 +45,19 @@ public void putAllServices(@NotNull Map<String, Service> services) {
         return policies;
     }
 
-    public void putAllPolicies(@NotNull Map<String, Policy> policies) {
-        this.policies.putAll(policies);
+    public synchronized void putAllPolicies(@NotNull Map<String, Policy> policies) {
```

Comment:
We could have a minor optimization here if we set policies as a concurrentHashMap and use the ConcurrentHashmap#merge function where we need to update the policies map. That way the updates are only locked per key instead of per map